lib/config/virtual_container_host.go

```diff
@@ -125,6 +125,9 @@ type VirtualContainerHostConfigSpec struct {
 
 	// Allow custom naming convention for containerVMs
 	ContainerNameConvention string
+
+	// default size for root image of image B tree
+	ScratchSize int `vic:"0.1" scope:"read-only" key:"scratch_size"`
```

Comment:
Do you intend for the size to be negative ever?  You may also want to mention the units expected.